@kiva/kv-components/vue/KvToast.vue

```diff
@@ -71,6 +71,10 @@
 							tw-flex
 						"
 						>
+							<slot
+								v-if="hasParentContentSlot"
+								name="parentContent"
+							></slot>
 							<!-- eslint-disable vue/no-v-html -->
 							<p
```

Comment:
The slot won't replace this message content entirely? Like either the component displays the slot or the old simple message string.

Reply:
There are other usages with the message prop, in that case we need to propagate this update to them or what do you suggest?

Reply:
I just meant that do we have a usage that needs _both_ the slot and the message prop? I was wondering if we needed a `v-else` on the `p` tag here.

Reply:
good call! as the slot is new, we gonna suppose that there is not a usage that needs both, ill add the `v-else` to p element as suggested, ty